packages/trie/examples/staticCreate.ts

```diff
@@ -0,0 +1,11 @@
+import { Trie } from '@ethereumjs/trie'
```

Comment:
What is the difference between this and `basicUsage.ts`?  Seems duplicative and maybe we can just remove this one?

Reply:
fixed